script/playbackstrategy/msestrategy.js

```diff
@@ -108,8 +109,11 @@ define('bigscreenplayer/playbackstrategy/msestrategy',
       }
 
       function setUpMediaElement (playbackElement) {
-        videoElement = document.createElement('video');
-
+        if (mediaKind === MediaKinds.AUDIO) {
+          videoElement = document.createElement('audio');
```

Comment:
Can we rename `videoElement` to `mediaElement` or similar? It's a bit misleading otherwise.